tests/testValues.ts

```diff
@@ -112,3 +112,17 @@ export const test_tattooUint8Array = new Uint8Array([
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.   
+    Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.   
+    Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi.   
+    Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo
+    Alle Unicode-Symbole und ihre Codes auf einer Seite, 𐎰, 103B0, 𐎱, 103B1, 𐎲, 103B2, 𐎳, 103B3, 𐎴, 103B4, 𐎵, 103B5, 𐎶, 103B6, 𐎷, 103B7, 𐎸, 103B8, 𐎹, 103B9, 𐎺, 103BA, 𐎻, 103BB, 𐎼, 103BC, 𐎽, 103BD, 𐎾, 103BE, 𐎿, 103BF, 𐏀, 103C0, 𐏁, 103C1, 𐏂, 103C2, 𐏃, 103C3, 103C4, 103C5, 103C6, 103C7, 𐏈, 103C8, 𐏉, 103C9, 𐏊, 103CA, 𐏋, 103CB, 𐏌, 103CC, 𐏍, 103CD, 𐏎, 103CE, 𐏏`;
```

Comment:
For safety this unicode should always be escaped - ie, don't trust the editor so the first should be `\u103B0` etc

Reply:
Wouldn't change that the test case? 
Or will every system deal with `\u103B0` the same way it deals with 𐎰?

Reply:
Javascript itself decodes it into real unicode internally when reading the string - but while we're probably all using editors that handle unicode internally properly there's no guarantee of that (or even that there isn't some global `.git` option for someone else that stops apparent text files from having the upper bit cleared as it's ascii etc) - being defensive is better (and also reads far easier in case someone doesn't have a unicode compatible font)